packages/augur-artifacts/src/addresses.json

```diff
@@ -81,32 +81,30 @@
   "TradingEscapeHatch": "0xc4d494c838f2ef079938453955cc7c7a71c4aac5"
  },
  "42": {
-  "Augur": "0x62214e5c919332AC17c5e5127383B84378Ef9C1d",
-  "Universe": "0x4d86aA1885c0f65bF2B4fe5D53C18a7E4264f5e1",
-  "LegacyReputationToken": "0x3B4c283A78F683F5c7C7089e20003498994f6726",
-  "Cash": "0x0802563FB6CfA1f07363D3aBf529F7b3999096f6",
-  "ProxyFactory": "0x47EA52e821d091d590d5E75c15d0Ac3C6eC0f2eF",
-  "GnosisSafe": "0x4338a7bdc83128ebaCA36bA43Db5e5e3eF8BdDcB",
-  "CashFaucet": "0x0802563FB6CfA1f07363D3aBf529F7b3999096f6",
-  "BuyParticipationTokens": "0x4b6Ac158E6cF9b1D194ef66D58c5Cd34376bC8dF",
-  "RedeemStake": "0x280AE885D2d61a956f1b5877c4872851A0241e54",
-  "GnosisSafeRegistry": "0x482677b8D15a4522C37a4748a346A2207c4613ff",
-  "WarpSync": "0xf7C06954DbB7128FF81abE9401b1BCCD10F02294",
-  "ShareToken": "0x63cbfEb0Cf1EE91Ca1689d8dbBa143bbf8Fd0fd1",
-  "HotLoading": "0xd2b29aC94303d9fa25a388dcA5E60f18C5737Aa1",
-  "Affiliates": "0xBeB5d056570d126c7ee00C7b6Ba999BC195409Cd",
-  "AffiliateValidator": "0x57D85FdFE6Cc549cab8Bc5ce7239Cb08beCD1347",
-  "Time": "0xF90CA7D8D124EFA886b3A44007270e0D2Bb80F38",
-  "AugurTrading": "0xcb5cf0c3d4e6604936DA3a46836B06C88390FCb8",
-  "CancelOrder": "0x32f2e0a65F102A9b17db441CC78394182031Ec4E",
-  "CreateOrder": "0xcE1AdC14365cC137d85Fa4De05E293828E7700d4",
-  "FillOrder": "0xa3c0b381dc9698D306e5Fedd914A0C47c236AD76",
-  "Orders": "0xF5275Fd4858376f103b191d36eD5ea77a737A3ed",
-  "ProfitLoss": "0xEcd2d609eE6fa584DfBA55C0A51c80E07F9E24df",
-  "SimulateTrade": "0xA9601aB48656f968aA6e2cf5A6a050589930037c",
-  "Trade": "0x294c9d1eB05b4Ed18b1e4e647B4551871bc448A1",
-  "ZeroXTrade": "0x99a6e4750BBAE1Eb6Cd7c6bC2A8633F10Af3a7D3",
-  "Exchange": "0x30589010550762d2f0d06f650d8e8b6ade6dbf4b",
```

Comment:
@robert-e-davidson3 @petong The below should also be included in the new deploy right? 
```
  "Exchange": "0x30589010550762d2f0d06f650d8e8b6ade6dbf4b",	
  "ZeroXExchange": "0x30589010550762d2f0d06f650d8e8b6ade6dbf4b"
```